### ISSUE TITLE
[Bug] Mod conflict (Spectra and Green Skull Revived)

### ISSUE THREAD

**Issue by bofhgr** [KSP-CKAN/NetKAN#7662]
Background
----------
* Operating System: latest win10 
* CKAN Version: latest
* KSP Version: latest

**Have you made any manual changes to your GameData folder (i.e., not via CKAN)?**
no

Problem
-------
**Describe the bug**
The mods Spectra and Green Skull Revived should not have a conflict, yet the installation fails.

**Steps to reproduce**
Install either mod first, then the other, see the error as per the error code below

**Expected behavior**
both mods should install together, there should not be incompatibility (?)

**Screenshots (if applicable)**

**CKAN error code (if applicable):**

```
About to install...

 * Green Skull Revived 1.0.1 (cached)

Oh no! We tried to overwrite a file owned by another mod!
Please try a `ckan update` and try again.

If this problem re-occurs, then it maybe a packaging bug.
Please report it at:

https://github.com/KSP-CKAN/NetKAN/issues/new

Please including the following information in your report:

File      : GameData/TextureReplacer/Default/EVAvisor.dds
Installing Mod : GreenSkullRevived 1.0.1
Owning Mod: Spectra
CKAN Version   : v1.26.10
Your GameData has been returned to its original state.
Error during installation!
An unknown error occurred, please try again!

```

**Comment by HebaruSan** [comment]
Might be a file included in Spectra by accident. "EVA visor" sounds like something that belongs in a suit pack, but not a planet enhancement pack.

**Comment by bofhgr** [comment]
I made a cross post to this bug to the respective mod owners on ksp forums, not sure if it's a CKAN issue but decided to post here as well. thanks!

**Comment by HebaruSan** [comment]
They look pretty different:

![image](https://user-images.githubusercontent.com/1559108/73809089-f3a69280-47c9-11ea-9c1c-69213dea8f9c.png)
(left is Spectra, right is GreenSkull)

So there appears to be a functional conflict here after all; Spectra will make your EVA visors look one way, Green Skull will make them look another way.

**Comment by bofhgr** [comment]
OK so I've looked into this a bit myself. Issue was both on Spectra mod and how it's configured on CAN. Suggestions *aka* I figured out the following: 
- Spectra should have Kopernicus as a Supported Mod on CKAN (not a requirement)
- Spectra should be a selection for both Environmental Visual Enhancements Config Files and Scatterer Config Files as well as the Sunflare component for Scatterer during installation via CKAN and have these mods as a Requirement
- Spectra should have Green Skull Revived as a conflicting mod for now (that EVA visor is not mentioned by the mod owner but it does conflict) until the mod owner either removes that file or confirms the conflict and mentions the visor in the installation notes (see below)

Now besides the above, I am unfamiliar with Texture Replacer and how it's supposed to be handled via CKAN with other mods so not sure to confirm whether it's actually required or not for that mod and if CKAN handles that part correctly.
